mlc_config.json

```diff
@@ -1,7 +0,0 @@
-{
```

Comment:
If we could use this instead of comment suppressing links checker in changes file, it would be nice. Users do not have to  forget about that comment.